subprojects/core/src/main/java/org/gradle/StartParameter.java

```diff
@@ -90,6 +81,7 @@
     private boolean continuous;
     private List<File> includedBuilds = new ArrayList<File>();
     private boolean buildScan;
+    private String bootstrapInitScriptUrl;
```

Comment:
I see what you mean now. We could potentially change the list to contain `Object`s which are evaluated later using the resolver. That way you could also use `--init-script someUrl` when you want to quickly test something. I'll let the others chime in on this too.